helix-core/src/syntax/overlay/test.rs

```diff
@@ -0,0 +1,32 @@
+use crate::syntax::{overlapping_overlay, Highlight, HighlightEvent::*};
+use std::iter;
+
+/// this tests checks that merging two overlapping ranges onto each other
+/// correctly preveres the order of merges.
+/// that is the highlight that is merged in last, gets applied last and overwrites the other layers
+/// In this test a range of lower priority (like a hint) starts at 2
+/// and another range of a high priority range (like an error) starts earlier
+/// with the old span implementation the hint would always overwrite the error.
+/// The new implementation (tested here) forces the
```

Comment:
Looks like this sentence got chopped off

Reply:
fixed in 87f1a17